src/types/index.ts

```diff
@@ -667,7 +667,6 @@ export type MoveUint128Type = string;
 export type MoveUint256Type = string;
 export type MoveAddressType = string;
 export type MoveObjectType = string;
-export type MoveStructType = `${string}::${string}::${string}`;
 export type MoveOptionType = MoveType | null | undefined;
 /**
  * String representation of a on-chain Move struct type.
```

Comment:
@xbtmatt Description will live here

Reply:
I think we should say something like...

```typescript
/**
* This is the format for a fully qualified struct, resource, or entry function in Move.
*/
```

Above it or something